static/app/views/insights/http/alerts.ts

```diff
@@ -0,0 +1,27 @@
+import type {AlertConfig} from 'sentry/views/insights/common/components/chartPanel';
+
+export const ALERTS: Record<string, AlertConfig> = {
+  spm: {
+    aggregate: 'spm()',
+    query: 'span.module:http span.op:http.client',
```

Comment:
This query looks the same across all alerts, I think it would be good if we create a shared variable for it.